migrations/041_import_jobs_status_check.sql

```
@@ -0,0 +1,116 @@
+-- `PRAGMA foreign_keys = off`); failed_imports carries a
+-- `REFERENCES import_jobs(id) ON DELETE SET NULL` that would otherwise block
+-- the drop.
+PRAGMA defer_foreign_keys = true;
+
+-- defer_foreign_keys postpones constraint *violation* checks to commit time,
+-- but it does not suppress ON DELETE actions: DROP TABLE import_jobs still
+-- fires SET NULL across failed_imports.import_id and silently severs every
+-- failure record from the job it describes. Snapshot the linkage here and
+-- restore it after the rename.
+CREATE TABLE import_jobs_fk_backup AS
+  SELECT id AS failed_import_id, import_id FROM failed_imports WHERE import_id IS NOT NULL;
```

SQLFluff (4.3.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)